LICENSE

```diff
@@ -0,0 +1,376 @@
+Copyright (c) 2020 HashiCorp, Inc.
```

Comment:
I don't think the license should be hashi copyrighted (but i am not a lawyer here!)